src/main/java/com/mtb/demo/dto/ColorDto.java

```diff
@@ -0,0 +1,10 @@
+package com.mtb.demo.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder
+@Getter
+public class ColorDto {
```

Comment:
Let's have record for dto without @Getter annotation, but leave @Builder annotation. Record for dto is a good practice. Make those changes for all dto's. Thanks

Reply:
public record ...., same for other dto's